build_system/src/build.rs

```diff
@@ -220,8 +225,11 @@ fn build_codegen(args: &mut BuildArg) -> Result<(), String> {
     })?;
 
     println!("[BUILD] sysroot");
```

Comment:
Please move this line inside the `if`.

Reply:
Alright, could be the reason why the CI is failing